### PR TITLE
Cleanup ss constraints

### PR DESCRIPTION
**Pull request by jallamsetty1**

### CODE REVIEW

File: modules/RTC/ScreenObtainer.js [hunk 1]
```diff
@@ -216,33 +216,30 @@ const ScreenObtainer = {
...
         }
 
-        const { desktopSharingFrameRate } = this.options;
-        const setScreenSharingResolutionConstraints = browser.isChromiumBased()
-            && this.options?.testing?.setScreenSharingResolutionConstraints;
+        const audio = this._getAudioConstraints();
         let video = {};
-
-        // Allow users to seamlessly switch which tab they are sharing without having to select the tab again.
-        if (browser.isChromiumBased() && browser.isVersionGreaterThan(106)) {
```
Comment (hristoterezov) [inline modules/RTC/ScreenObtainer.js:225]: Why are we removing this?

File: modules/RTC/ScreenObtainer.js [hunk 2]
```diff
@@ -216,33 +216,30 @@ const ScreenObtainer = {
...
+        if (browser.isChromiumBased()) {
+            // Allow users to seamlessly switch which tab they are sharing without having to select the tab again.
+            browser.isVersionGreaterThan(106) && (video.surfaceSwitching = 'include');
 
-        // At the time of this writing 'min' constraint for fps is not supported by getDisplayMedia.
-        video.frameRate && delete video.frameRate.min;
+            // Set bogus resolution constraints to work around
+            // https://bugs.chromium.org/p/chromium/issues/detail?id=1056311 for low fps screenshare. Capturing SS at
+            // very high resolutions restricts the framerate. Therefore, skip this hack when capture fps > 5 fps.
+            if (desktopSharingFrameRate?.max <= SS_DEFAULT_FRAME_RATE) {
```
Comment (hristoterezov) [inline modules/RTC/ScreenObtainer.js:238]: It seems before we were applying this for all browsers but now only for chromium based. This is intentional and it is chrome specific?

File: modules/RTC/TraceablePeerConnection.js [hunk 3]
```diff
@@ -3104,8 +3094,7 @@ TraceablePeerConnection.prototype.generateNewStreamSSRCInfo = function(track) {
     }
 
     // Configure simulcast for camera tracks and desktop tracks that need simulcast.
-    if (this.isSimulcastOn()
-        && (track.getVideoType() === VideoType.CAMERA || !this.isSharingLowFpsScreen())) {
```
Comment (hristoterezov) [inline modules/RTC/TraceablePeerConnection.js:3108]: Why do we remove the `!this.isSharingLowFpsScreen()`?

Also probably the comment needs an update and also the braces are redundant.

**Review by hristoterezov — CHANGES_REQUESTED** [review]
Left some questions and comments!

File: modules/RTC/ScreenObtainer.js (hunk 2 above)
Comment (jallamsetty1) [reply modules/RTC/ScreenObtainer.js:238]: It is Chrome specific and before also we had the Chrome check - https://github.com/jitsi/lib-jitsi-meet/pull/2195/files#diff-ce2d682b5181dd9a7081e3a971c936cf99e8c6d24644c7dafcd0ee10e85d30f5L220

File: modules/RTC/TraceablePeerConnection.js (hunk 3 above)
Comment (jallamsetty1) [reply modules/RTC/TraceablePeerConnection.js:3108]: This is part of plan-b flow and not executed anymore. I will do another PR for cleaning up plan-b stuff.

Comment (jallamsetty1) [reply modules/RTC/TraceablePeerConnection.js:3108]: If you prefer, I can leave this in for now.

**Comment by jallamsetty1** [comment]
Jenkins test this please.

**Review by hristoterezov — APPROVED** [review]

**Pull request merged**